server/src/sessionManagement.mjs

```diff
@@ -31,13 +32,16 @@ class FireboltCall {
     }
 }
 
-class Session{
+class Session {
     #sessionStart;
     #sessionEnd;
     
     constructor(){
         this.calls = [];
         this.#sessionStart = Date.now();
+        this.sessionOutput = "log";
```

Comment:
I thought we talked about allowing a user to do as many of the output formats at a time as they want? This would need to be an array then, and the CLI param would have to be multi-value.

Reply:
I remember discussing the possibility of saving multiple files by default but don't think we specifically mentioned the user being able to specify multiple outputs. In an effort to get this ticket closed can we leave it as is and add an enhancement ticket to allow this option if desired?

Reply:
Yes... I think separating that as a distinct ticket is a good idea.